lib/tasks/update_pdf_yml_buttons.rake

```diff
@@ -0,0 +1,152 @@
+desc "Update application yml mappings for pdf files with button options"
```

Comment:
This is used for development only - it updates the pdf yaml files with the latest from the pdf file - intended to be used as a 'starting point' not the final yaml version - use with caution